src/diffusers/models/resnet.py

```diff
@@ -97,7 +247,15 @@ def __init__(
         conv_2d_out_channels: Optional[int] = None,
     ):
         super().__init__()
-        self.pre_norm = pre_norm
+        if time_embedding_norm == "ada_group":
+            raise ValueError(
+                "Passing `ada_group` as `time_embedding_norm` is deprecated, please create `ResnetBlockCondNorm2D` instead",
+            )
```

Comment:
I am not sure if saying something is deprecated is a good idea in a "ValueError". Have we ever done that?

Reply:
Yeah, let's indeed rename the message here to something like "This class cannot be used with "type==ada_group", please use XXX instead"

Reply:
updated!